### ISSUE TITLE
Accessibility: NVDA emits errors in WT1.15 preview

### ISSUE THREAD

**Issue by leonardder** [microsoft/terminal#13866]
### Windows Terminal version

1.15.2283.0

### Windows build number

25188.1000

### Other Software

NVDA 2022.2.2

### Steps to reproduce

1. Open bash in windows terminal
2. type nano /etc/hosts

### Expected Behavior

No error in NVDA

### Actual Behavior

NVDA emits the following error:

```
ERROR - NVDAObjects.behaviors.LiveText._monitor (18:21:06.798) - Dynamic_WinTerminalUIAEditableTextWithAutoSelectDetectionUIA._monitorThread (22344):
Error getting or calculating new text
Traceback (most recent call last):
  File "NVDAObjects\behaviors.pyc", line 386, in _monitor
  File "NVDAObjects\behaviors.pyc", line 350, in _getText
  File "diffHandler.pyc", line 62, in _getText
  File "baseObject.pyc", line 26, in __get__
  File "NVDAObjects\UIA\__init__.pyc", line 848, in _get_text
  File "NVDAObjects\UIA\__init__.pyc", line 555, in _getTextFromUIARange
  File "monkeyPatches\comtypesMonkeyPatches.pyc", line 32, in __call__
_ctypes.COMError: (-2147467259, 'Unspecified error', (None, None, None, 0, None))
```

I wasn't able to reproduce this with WT1.14, so therefore I'm filing this here since there might be a regression in UIA related code. @codeofdusk could you reproduce this?

**Comment by codeofdusk** [comment]
I've also seen this, but not with `wtNotifications`.

This is probably related to broken assumptions when switching to/from the alt buffer.

**Comment by leonardder** [comment]
This error indeed has nothing to do with the notifications implementation.

CC @carlos-zamora just want to make sure that this bug doesn't end up in a release, since that would be annoying for many sr users.

**Comment by carlos-zamora** [comment]
Yeah, that error code is `E_FAIL` so it's 100% that alt buffer thing.
